plugins/local_file/src/interface.h

```diff
@@ -0,0 +1,10 @@
+#ifndef USER_MAPPER_INTERFACE
+#define USER_MAPPER_INTERFACE
+
+extern "C" {
+  void* user_manager_init(const char* _args);
+  char* user_manager_match(void* _manager, const char* _param);
+  void user_manager_close(void* _manager);
+}
+
+#endif
```

Comment:
Needs trailing comment with include guard.

Reply:
This isn't clicking for me, can you give an example?

Reply:
```cpp
#endif // IRODS_HTTP_API_USER_MAPPING_HPP
```

Reply:
Done.